CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ All notable changes to this project will be documented in this file.  The format
 Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
+## 0.10.1 10-03-2024
```

Comment:
09-03-2024

I just noticed the below 0.10.0 date is also wrong

Reply:
Thanks. I am addressing this in a separate PR. I am also going to have to update this PR as more changes are required.